openpype/modules/royalrender/lib.py

```diff
@@ -175,6 +176,18 @@ def get_job(self, instance, script_path, render_path, node_name):
             instance, render_path, start_frame, end_frame)
         instance.data["expectedFiles"].extend(expected_files)
 
+        submitter_parameters = []
+
+        anatomy_data = instance.context.data["anatomyData"]
+        environment = RREnvList({
+            "AVALON_PROJECT": anatomy_data["project"]["name"],
+            "AVALON_ASSET": instance.context.data["asset"],
+            "AVALON_TASK": anatomy_data["task"]["name"],
+            "AVALON_APP_NAME": instance.context.data.get("appName"),
+            "AYON_RENDER_JOB": "1",
+            "AYON_BUNDLE_NAME": os.environ["AYON_BUNDLE_NAME"]
```

Comment:
This is breaking compatibility with OP

Reply:
This is in more places. Also I don't think it is adding rrEnv file to the job. But for god sake I can't find that information anywhere in the **rrControl** app... I thought it should be somewhere around here but it isn't:

![image](https://github.com/ynput/OpenPype/assets/33513211/d859df5d-433b-4c33-ad59-088d7dc74cc2)